src/main/java/com/akademia/projectplanner/repository/TaskRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.akademia.projectplanner.repository;
+
+import com.akademia.projectplanner.model.Task;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface TaskRepository extends JpaRepository<Task, Long> {
+
+    boolean existsByName(String name);
```

Comment:
good practice: nie piszemy "na zapas/przyszłość" - jeśli metoda będzie potrzebna to się ją dopiero wtedy zaimplementuje